webapp/components/reviewOperation/amount.tsx

```diff
@@ -15,11 +16,17 @@ export const Amount = function ({ token, value }: Props) {
   return (
     <div className="flex items-center justify-between text-sm font-medium">
       <span className="text-neutral-500">{t('total-amount')}</span>
-      <div className="text-neutral-950">
-        <DisplayAmount
-          amount={formatUnits(BigInt(value), token?.decimals ?? 18)}
-          token={token}
-        />
+      <div className="flex gap-x-1">
+        <div className="text-neutral-950">
+          <DisplayAmount
+            amount={formatUnits(BigInt(value), token?.decimals ?? 18)}
+            token={token}
+          />
+        </div>
+        <div className="flex items-center gap-x-1 font-normal text-neutral-500">
+          <span>($</span>
+          <FiatBalance token={token} />)
```

Comment:
Shouldn't encapsulate this logic? Wouldn’t something like this be better?
```suggestion
          <span>(${<FiatBalance token={token} />})</span>
```